### DISCUSSION TITLE
TESTNET

### DISCUSSION THREAD

**Discussion by Georgeedet** [Cyfrin/blockchain-basics-cu#2] [Q&A]
Hello, I need help funding my testnet wallet. I've tried Chainlink Faucet and it's requesting I have 0.001 ETH. Other faucets I've tried also have similar issues.

**Comment by KMean** [comment]
hey @Georgeedet try one of these:

[google cloud](https://cloud.google.com/application/web3/faucet/ethereum/sepolia)
[Sepolia POW Faucet](https://sepolia-faucet.pk910.de/)

**Reply by codex-scribe** [reply]
Thank you!  Yes, the course notes should be updated. **Google cloud works;** the other faucets require the user to have .001 ETH on mainnet and there doesn't seem to be any way around it.  @PatrickAlphaC

**Reply by PatrickAlphaC** [reply]
We are going to be moving to using virtual testnets. These ones have become too challenging. Sorry all.

**Reply by codex-scribe** [reply]
Maybe there's a lot of developers working on sepolia now?  I noticed that the blocks have >100 transactions in them.  

I finally just broke down and sent 1 LINK mainnet to my wallet so I could get these tokens.  It's felt a lot easier since then.  But 1 link costs about $20, and then you have to pay for an ETH transaction to send it.

One other thought I had was it seemed like one of the faucets was run by alchemy.  They wanted you to create a free account and then you might be eligible for more funds from their faucet.  I didn't do that at the time because I didn't want to create any new accounts just for that, but now that we are in the foundry part of the course, I have created an alchemy account anyway.  So there might be a faucet there that it makes sense for people to use.  And you can say in the earlier part of the course about faucets that we will eventually be using alchemy, so it may make sense to create an account.

**Reply by PatrickAlphaC** [reply]
We are updating to use virtual networks very soon. The testnet deployment is a "nice to have" because it lets you "feel" what the process is like, but it's not 100% required!
